ares/n64/pi/dma.cpp

```diff
@@ -40,8 +39,9 @@ auto PI::dmaWrite() -> void {
       cur_len = max(cur_len-misalign, 0);
     }
 
+    cpu.recompiler.invalidateRange(io.dramAddress, io.dramAddress + cur_len);
```

Comment:
`if constexpr(Accuracy::CPU::Recompiler) {`

As an aside, the existing invalidate() calls in Bus::write() should be guarded by this check, too. All other references to the recompilers in the n64 core are guarded this way.

Reply:
Done